doc/api/deprecations.md

```diff
@@ -1192,7 +1194,7 @@ changes:
 Type: End-of-Life
 
 The `util.isNullOrUndefined()` API has been removed. Please use
-`arg === null || arg === undefined` instead.
+`arg == null` instead.
```

Comment:
The previous suggestion was more correct, if `arg` is an instance of `HTMLAllCollection`, `arg == null` would be `true` while ``util.isNullOrUndefined(arg)` and `arg === null || arg === undefined` would both be `false`.

Reply:
proposed here https://github.com/nodejs/node/pull/59269#discussion_r2240503662

Reply:
should I revert this change ?

Reply:
https://github.com/nodejs/node/pull/59269#discussion_r2240503662 suggests this because your original change was even more wrong (i.e. it would not treat `undefined` as a false negative). This new suggestion still has a blind spot (it treats `document.all` as a false positive), IMO yes reverting is the correct call, because this change is not an improvement.

FWIW you can find this behavior spec'd in step 4 of the [`IsLooselyEqual`](https://tc39.es/ecma262/#sec-islooselyequal) section of ECMAScript. Node.js users should encounter it only with Electron or similar.

Reply:
ah, right. that's a fair point. in regular node code there's no such thing as `document.all` so it has no impact, but as a general JS guideline and in electron, @aduh95 is correct.

Reply:
i had reverted change